go.mod

```diff
@@ -23,7 +25,7 @@ require (
 	github.com/sirupsen/logrus v1.5.0
 	github.com/spf13/cobra v0.0.7
 	github.com/spf13/viper v1.6.3
-	github.com/syndtr/goleveldb v1.0.0
+	github.com/syndtr/goleveldb v1.0.1-0.20190923125748-758128399b1d
```

Comment:
I do not think that this change is relevant for this PR. I would keep dependency upgrades in separate PRs, unless this is something that go tool is insisting to change.

Reply:
I'll edit go.mod. No idea why they're there.

Reply:
There seems to be some version-neutral includes here:

```
	golang.org/x/crypto v0.0.0-20200406173513-056763e48d71
	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
```

should we nail down a version for them?

Reply:
hmm these things don't even seem to _have_ versioning.